apps/style/craft/style.scss

```diff
@@ -546,11 +546,12 @@ $height-to-width: math.div(434, 477);
     @include pixelated;
     @include gray-button;
     @include mc-text-shadow;
+    @include main-font-semi-bold;
 
+    font-family: $main-font !important;
```

Comment:
Why do we need both of these? I'm guessing the !important would override right?

Reply:
`!important` is here to create the same priority scss rule (before the change of fontFamily `!important` was already here)

I've also used mixin cause right now we also need to set font weith and font styles alongside with font family